supplements/extra-life/one-grung-above.xml

```diff
@@ -98,7 +98,7 @@
 			<p>You’re immune to poison damage and the poisoned condition.</p>
 		</description>
 		<sheet>
-			<description>You’re immune to poison damage and the poisoned condition.</description>
+			<description>You are immune to the poisoned condition.</description>
```

Comment:
Why is this being changed?  The rules are still granting immunity to poison damage and JC has confirmed that immunity to poison is both damage and the condition.  Checked DDB and it still has old text.

Reply:
because it has a granted immunity to damage. There's no reason to have that part in sheet description.